src/components/Navbar/Navbar.js

```diff
@@ -1,27 +1,29 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
+import { I18N } from '../../utils/constants';
+
 import style from './Navbar.module.css';
 
 const Navbar = () => (
   <nav className={style.nav}>
     <div className={style.item}>
-      <NavLink to="/profile" activeClassName={style.activeLink}>Profile</NavLink>
+      <NavLink to="/profile" activeClassName={style.activeLink}>{I18N.EN.PROFILE}</NavLink>
```

Comment:
Consider adding these nav link destinations ("to" prop values) to ROUTES object.